src/types/api/nodes.ts

```diff
@@ -103,12 +103,13 @@ enum EConfigState {
 }
 
 export enum ESort {
-    NodeId,
-    Host,
-    DC,
-    Version,
-    Uptime,
-    Memory,
-    CPU,
-    LoadAverage,
+    NodeId = 'NodeId',
+    Host = 'Host',
+    DC = 'DC',
+    Rack = 'Rack',
```

Comment:
Added `Rack`. Now this `ESort` is the same as `ESort` for `/nodes`, but I decided to have two different enums since endpoints are different and they are handled differently